### DISCUSSION TITLE
Get directly the "Name" not record issue

### DISCUSSION THREAD

**Discussion by MoFlow00** [tmijail/Odoo-Power-BI-Connector#25] [Q&A]
How can i get directly the "name" instead of "Record" because when it's return with record and using the extend in power query it's reload again

**Comment by tmijail** [comment]
You can't. The reload only happens in the preview, though. It has no impact on the performance of refreshes.

**Reply by MoFlow00** [reply]
Okay thank you . It's working perfectly! 

I have in my data more than 60 columns i need to fetch only 10 columns. Can i do that ?

**Reply by tmijail** [reply]
Yes.

In version 1.0.0 of the connector you need to use the `search_read` function. See [this part of the readme](https://github.com/tmijail/Odoo-Power-BI-Connector#search_read).

Alternatively, you can use version [1.1.0 RC](https://github.com/tmijail/Odoo-Power-BI-Connector/releases/tag/v1.1.0-rc) and just use the query editor UI to delete the unwanted columns.

**Reply by MoFlow00** [reply]
Thank you! it's working perfectly!

Also, I have modified the // split them into columns row to return with the "name" instead of "ID" & "Name"  and it's working perfectly!

list_to_record_map = (x) => if (x <> null) then x{1} else null
